plugins/generator-1.15.2/forge-1.15.2/templates/gui.java.ftl

```diff
@@ -478,6 +480,11 @@ import ${package}.${JavaModName};
 						handleButtonAction(entity, ${btid}, x, y, z);
 					}));
 					<#assign btid +=1>
+			    <#elseif component.getClass().getSimpleName() == "Checkbox">
+            	    ${component.name} = new CheckboxButton(this.guiLeft + ${(component.x - mx/2)?int}, this.guiTop + ${(component.y - my/2)?int},
+            	        150, 20, new StringTextComponent("${component.text}"), ${component.isChecked});

```

Comment:
Looks like there's a problem here. It takes `String` as a parameter but you only put `StringTextComponent`